scripts/clatd@.systemd

```diff
@@ -0,0 +1,22 @@
+#
+# clatd service file for systemd
+#
+# Install it to:  /etc/systemd/system/clatd.service
+# Enable it with: systemctl enable clatd.service
+# Start it with:  systemctl start clatd.service
+# 
+# Written by Tore Anderson <tore@fud.no>
+#
+
+[Unit]
+Description=464XLAT CLAT daemon
+Documentation=man:clatd(8)
+After=network-online.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/clatd -c /etc/clatd/%i plat-dev=%i
```

Comment:
Maybe this should have been `/etc/clatd/%i.conf` instead?

Reply:
done